docs/advanced/advanced_tech_info.md

```diff
@@ -0,0 +1,617 @@
+              - name: demo-volume-raw-block
+                devicePath: "/dev/block"**
+          volumes:
+          - name: demo-volume-raw-block
+            persistentVolumeClaim:
+              claimName: demo-pvc-raw-block
+    ​
+    #      nodeSelector:
+    #        kubernetes.io/hostname: HOSTNAME
+    ```
+
+6.  Check the newly created pod.
+
+    Display the newly created pod \(make sure the pod status is Running\).
+
+    ```
+    kubectl get pod demo-statefulset-raw-block-0
+    NAME                 READY   STATUS    RESTARTS   AGE
+    demo-statefulset-raw-block-0   1/1     Running   0          43s
+    ```
+
+7.  Write data to the persistent volume of the pod.
+
+    The PV should be mounted inside the pod at /dev.
+
+    ```
+    kubectl exec demo-statefulset-raw-block-0 -- bash -c " echo "test_block" | dd conv=unblock of=/dev/block"
+    0+1 records in
+    0+1 records out
+    11 bytes copied, 9.3576e-05 s, 118 kB/s
+    
+    kubectl exec demo-statefulset-raw-block-0 -- bash -c "od -An -c -N 10 /dev/block"
+    t e s t _ b l o c k  
+    ```
+
+8.  Delete StatefulSet and then recreate, in order to validate data \(test\_block in /dev/block\) remains in the persistent volume.
+
+    1.  Delete the StatefulSet.
+
+        ```screen
+        $> kubectl delete statefulset/demo-statefulset-raw-block
+        statefulset/demo-statefulset-raw-block deleted
+        ```
+
+    2.  Wait until the pod is deleted. Once deleted, the '"demo-statefulset-file-system" not found' is returned.
+
+        ```screen
+        $> kubectl get statefulset/demo-statefulset-raw-block
+        Error from server (NotFound): statefulsets.apps <StatefulSet name\> not found
+        ```
+
+    3.  Recreate the StatefulSet and verify that the content written to /dev/block exists.
+
+        ```screen
+        $> kubectl create -f demo-statefulset-raw-block.yaml
+        statefulset/demo-statefulset-raw-block created
+        
+        $\> kubectl exec demo-statefulset-raw-block-0 -- bash -c "od -An -c -N 10 /dev/block"
+        t e s t \_ b l o c k 
+        ```
+
+9.  Delete StatefulSet and the PVC.
+
+    ```screen
+    $> kubectl delete statefulset/demo-statefulset-raw-block
+    statefulset/demo-statefulset-raw-block deleted
+    
+    $> kubectl get statefulset/demo-statefulset-raw-block
+    No resources found.
+    
+    $> kubectl delete pvc/demo-pvc-raw-block
+    persistentvolumeclaim/demo-pvc-raw-block deleted
+    
+    $> kubectl get pv,pvc
+    No resources found.
+    ```
+
+
+## Troubleshooting
+
+### Log collection for CSI pods, daemonset, and StatefulSet
+
+```
+kubectl get all -n *<namespace\>*  -l csi
+```
+
+For example:
+
+```screen
+$> kubectl get all -n *<namespace\>* -l csi
+NAME READY STATUS RESTARTS AGE
+pod/ibm-block-csi-controller-0 6/6 Running 0 2h
+pod/ibm-block-csi-node-nbtsg 3/3 Running 0 2h
+pod/ibm-block-csi-node-wd5tm 3/3 Running 0 2h
+pod/ibm-block-csi-operator-7684549698-hzmfh 1/1 Running 0 2h
```

Comment:
was the column spacing here unaligned in the first place?

Reply:
Yes but i fixed it now

Reply:
still doesn't like aligned to me, but I guess that's how it always has been

Reply:
if you are able to play with the MD file directly, feel free to do so for this specific item. 
Yes, it's always been like this though. It looked like it would work in VSC and in its preview